rasa_core/tracker_store.py

```diff
@@ -192,6 +194,13 @@ def save(self, tracker, timeout=None):
 
     def retrieve(self, sender_id):
         stored = self.conversations.find_one({"sender_id": sender_id})
+
+        # look for conversations which have used an `int` sender_id in the past and update them.
+        if stored is None and sender_id.isdigit():
+            stored = self.conversations.find_one_and_update({"sender_id": int(sender_id)},
```

Comment:
Just checking, this returns None if no such document is found right?

Reply:
I actually tried it out since the documentation of pymongo is about unclear about that.
![image](https://user-images.githubusercontent.com/7667273/47086248-421cd000-d219-11e8-9553-603a2ecbe955.png)